x-pack/plugins/security/public/management/management_service.ts

```diff
@@ -28,30 +27,26 @@ interface SetupParams {
   getStartServices: StartServicesAccessor<PluginStartDependencies>;
 }
 
-interface StartParams {
-  management: ManagementStart;
-}
-
 export class ManagementService {
   private license!: SecurityLicense;
   private licenseFeaturesSubscription?: Subscription;
+  private securitySection?: ManagementSection;
 
   setup({ getStartServices, management, authc, license, fatalErrors }: SetupParams) {
     this.license = license;
+    this.securitySection = management.sections.section.security;
 
-    const securitySection = management.sections.getSection(ManagementSectionId.Security);
-
-    securitySection.registerApp(usersManagementApp.create({ authc, getStartServices }));
-    securitySection.registerApp(
+    this.securitySection.registerApp(usersManagementApp.create({ authc, getStartServices }));
+    this.securitySection.registerApp(
       rolesManagementApp.create({ fatalErrors, license, getStartServices })
     );
-    securitySection.registerApp(apiKeysManagementApp.create({ getStartServices }));
-    securitySection.registerApp(roleMappingsManagementApp.create({ getStartServices }));
+    this.securitySection.registerApp(apiKeysManagementApp.create({ getStartServices }));
+    this.securitySection.registerApp(roleMappingsManagementApp.create({ getStartServices }));
   }
 
-  start({ management }: StartParams) {
+  start() {
     this.licenseFeaturesSubscription = this.license.features$.subscribe(async (features) => {
-      const securitySection = management.sections.getSection(ManagementSectionId.Security);
+      const securitySection = this.securitySection as ManagementSection;
```

Comment:
Why did you have to cast here?

Reply:
We know its been set in the `setup` method but typescript isn't aware of this. It thinks it might be undefined.

Reply:
Won't `this.securitySection!` work as well. I find it clearer.